src/category_theory/shift.lean

```diff
@@ -26,20 +26,21 @@ class has_shift :=
 
 variables [has_shift.{v} C]
 
-/-- The shift functor, moving objects and morphisms 'up'. -/
-def shift : C ⥤ C := (has_shift.shift.{v} C).functor
+/-- The shift autoequivalence, moving objects and morphisms 'up'. -/
+def shift : C ≌ C := has_shift.shift.{v} C
 
 -- Any better notational suggestions?
-notation X`[1]`:20 := (shift _).obj X
-notation f`[[1]]`:80 := (shift _).map f
+notation X`⟦`n`⟧`:20 := ((shift _)^(n : ℤ)).functor.obj X
+notation f`⟦`n`⟧'`:80 := ((shift _)^(n : ℤ)).functor.map f
 
-example {X Y : C} (f : X ⟶ Y) : X[1] ⟶ Y[1] := f[[1]]
+example {X Y : C} (f : X ⟶ Y) : X⟦1⟧ ⟶ Y⟦1⟧ := f⟦1⟧'
+example {X Y : C} (f : X ⟶ Y) : X⟦-2⟧ ⟶ Y⟦-2⟧ := f⟦-2⟧'
 
 open category_theory.limits
 variables [has_zero_morphisms.{v} C]
 
 @[simp]
-lemma shift_zero (X Y : C) : (0 : X ⟶ Y)[[1]] = (0 : X[1] ⟶ Y[1]) :=
+lemma shift_zero (X Y : C) (n : ℤ) : (0 : X ⟶ Y)⟦n⟧' = (0 : X⟦n⟧ ⟶ Y⟦n⟧) :=
```

Comment:
This name is now ambiguous, because there should also be a def `shift_zero : X[[0]] \isom X`...
Maybe we should call that one `shift_by_zero`?

Reply:
I've renamed this one to `shift_zero_eq_zero`.